cupyx/scipy/sparse/linalg/_interface.py

```diff
@@ -0,0 +1,679 @@
+import warnings
+
+import cupy as cp
```

Comment:
In our codebase, we don't use `cp` alias for readability.

Reply:
✌🏾  😅